data/_ui/interfaces.txt

```diff
@@ -1298,6 +1299,88 @@ interface "bank"
 
 
 
+interface "shop"
+	anchor bottom right
+	value "fleet bar width" 250
+	value "item bar width" 300
+	box "find button"
+		center -580 -20
+		dimensions 26 26
+	box "leave button"
+		center -45 -25
+		dimensions 70 30
+	box "park button"
+		center -125 -25
+		dimensions 70 30
+	box "buy button"
+		center -505 -25
+		dimensions 70 30
+	box "sell button"
+		center -425 -25
+		dimensions 70 30
+	box "sell ship button"
+		center -215 -25
+		dimensions 50 30
+
+	button f ""
+		center -580 -20
+		dimensions 26 26
+
+	button l "_Leave"
+		center -45 -25
+		dimensions 70 30
+		size 18
+
+	visible if "use buy"
+	active if "can buy"
+	button b "_Buy"
+		center -505 -25
+		dimensions 70 30
+		size 18
+
+	visible if "use install"
+	active if "can buy"
+	button b "_Install"
+		center -505 -25
+		dimensions 70 30
+		size 18
+
+	visible if "use cargo"
+	active if "can buy"
+	button b "_Cargo"
+		center -505 -25
+		dimensions 70 30
+		size 18
+
+	visible if "is outfitter"
+	active if "can sell"
+	button s "_Sell"
+		center -425 -25
+		dimensions 70 30
+		size 18
+
+	visible if "!is outfitter"
+	active if "can sell"
+	button s "_Sell"
+		center -215 -25
+		dimensions 50 30
```

Comment:
```suggestion
		dimensions 70 30
```

Reply:
I'm happy to make the Sell buttons match the other button size, but the existing width is 50px which I presume is to help avoid accidental clicks, so I didn't want to change that as part of this. Are you sure you want me to change this?

Reply:
Yes. It looks more awkward now that it has buttons on both sides, and we have a popup confirmation dialog that should help avoid accidentally selling ships.